app/views/backend/products/show.html.haml

```diff
@@ -40,6 +40,15 @@
           %span.value
             %i.picto{class: "picto-movements"}
             = link_to(resource.outgoing_parcel_item.parcel.number, controller: :parcels, action: :show, id: resource.outgoing_parcel_item.parcel.id)
+      .med-info.important
+        %span.title CUMP
```

Comment:
Same thing here as in `app/views/backend/product_nature_variants/show.html.haml`.